### PR TITLE
Publish conjure-java-core

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Conjure-Java ![Bintray](https://img.shields.io/bintray/v/palantir/releases/conjure-java.svg) [![License](https://img.shields.io/badge/License-Apache%202.0-lightgrey.svg)](https://opensource.org/licenses/Apache-2.0)
 
-
-
-
 _CLI to generate Java POJOs and interfaces from [Conjure API definitions](https://github.com/palantir/conjure)._
 
+Note that all Java APIs in this repository (including `conjure-java-core`) are considered "internal" in the sense that
```
Comment (uschi2000) [inline readme.md:5]: @iamdanfox @cakofony

**Comment by ferozco** [comment]
@uschi2000 can we close this out?

**Comment by uschi2000** [comment]
Yep. We’ll add PRs for undertow and dialogue.

**Comment by ellisjoe** [comment]
@iamdanfox

**Comment by iamdanfox** [comment]
I'm OK with revisiting this because the original motivation was to ensure any OSS build-tools don't rely on these java classes, but since we don't know of any external build tools anyway, we're getting all the downsides of friction in dialogue and none of the upsides.

:+1: from me.

**Pull request merged**